developer-docs-site/static/move_demo/sources/CAPMessageBoard.move

```diff
@@ -0,0 +1,141 @@
+module Examples::CapMessageBoard {
+    use Std::Vector;
+    use Std::Signer;
+    use CoreFramework::Timestamp;
+    use Std::Event::{Self, EventHandle};
+
+    // Error map
+    const ACCOUNT_NO_NOTICE_CAP: u64 = 0;
+    const CANNOT_REMOVE_ADMIN_NOTICE_CAP: u64 = 1;
+
+    struct CapMessageBoard has key, store {
+        notice: vector<u8>
+    }
+    /// delegate the notice update capability,
+    /// the reason having this struct is we cannot directly transfer capability across accounts
+    /// very unintuitive
+    struct NoticeChangeCapabilityContainer has key, store {
+        addr: address
```

Comment:
shouldn't this be a vector of addresses so we can distribute out the capability?

Reply:
We can still distribute the cap to multiple participants. 
Admin updates the addr. a participant subscribe the NoticeCapEvent will notice this change and claim the capability. 
when admin wants to add a new participant. admin updates the add again. A new participant can claim the capability. Now two participants have the capability to post to the message board. 
I feel this example can better showcase we don't need to maintain any vector to distribute the cap